resources/views/dashboard/controllers/events/view.blade.php

```diff
@@ -87,15 +87,52 @@
                                         </td>
                                         @if(Auth::user()->isAbleTo('events'))
                                             <td>
-                                                <span data-toggle="modal" data-target="#addrequest{{ $r->id }}">
-                                                    <button type="button" class="btn btn-success btn-sm simple-tooltip" data-placement="top" data-toggle="tooltip" title="Assign Position"><i class="fas fa-check"></i></button>
-                                                </span>
-                                                &nbsp;
-                                                <a href="/dashboard/controllers/events/view/{{ $r->id }}/un-signup" class="btn btn-danger btn-sm simple-tooltip" data-toggle="tooltip" title="Delete Request"><i class="fas fa-times"></i></a>
+
+                                                <div class="btn-group" role="group" aria-label="Actions">
```

Comment:
Button group looks nice - good work here matching the style of the rest of the admin pages